### PR TITLE
Update README (Addresses #110)

### PR DESCRIPTION
**Pull request by jakevdp**
A draft update for our README file.

### CODE REVIEW

**Comment by jakevdp** [comment]
TODO: add simple example + PNG

**Comment by ellisonbg** [comment]
Merge and iterate...

File: README.md [hunk 1]
```diff
@@ -27,26 +41,27 @@ There are currently many excellent plotting libraries in Python. The main ones a
...
-of two numerical columns, a scatterplot is almost certainly a good starting point. If you add a
-categorical column to that, you probably want facetted scatterplots. In cases where the
-visualization can't be inferred, simple user interfaces can enable the construction of
-visualizations without any coding. [Tableau](http://www.tableau.com/) and Jeff Heer's
+Another challenge is that all of the current APIs require the user to write code, even for the simplest of visualizations.
+This is unfortunate and unnecessary as the type of visualization (histogram, scatterplot, etc.) can often be inferred with basic information such as the columns of interest and the data types of those columns.
+For example, if you are interested in a visualization of two numerical columns, a scatterplot is almost certainly a good starting point.
+If you add a categorical column to that, you probably want facetted scatterplots.
+In cases where the visualization can't be inferred, simple user interfaces can enable the construction of visualizations without any coding. [Tableau](http://www.tableau.com/) and Jeff Heer's
```
Comment (kanitw) [inline README.md]: Interactive Data Lab's? :)

Comment (ellisonbg) [inline README.md]: Good catch, fixed in master.

**Pull request merged**